### DISCUSSION TITLE
Rewards for the top bug reporters October 2021

### DISCUSSION THREAD

**Discussion by FrancescoBorzi** [chromiecraft/chromiecraft#2241] [General]
This is the number of **VALID** bug reports per every contributor that have been validated in October 2021.
When a bug is validated, it gets the  [Linked [AC] label](https://github.com/chromiecraft/chromiecraft/issues?q=is%3Aissue+is%3Aopen+label%3A%22Linked+%5BAC%5D%22).

If your bug hasn't been validated yet, don't worry: it will be counted in a later month. Invalid bug reports will of course not be counted.

Source: https://www.chromiecraft.com/top-contributors-of-october-2021/

The people mentioned here: in order to receive your Chromie points, please send me a private message on Discord (`Shin#0632`) with the following:

- your GitHub username
- your game account username
- your game account email

Remember to give us a Star on our main GitHub repo: https://github.com/azerothcore/azerothcore-wotlk

@Shineslippers – 26
@TheRiddermark – 12
@Synful-Syn – 12
@Elexias – 12
@ntrna – 9
@Snufflenose – 9
@cachocarne – 5
@discorder0 – 4
@baranzadeoglu – 4
@ghost – 3
@Plan414 – 3
@Lomsikka – 3
@BobsenJr – 3
@vexima – 2
@slicplaya – 2
@phonkala – 2
@lokpoki – 2
@gurveleg – 2
@b0bzzter – 2
@Wtrflw – 2
@VFaleiro – 2
@PendingtonFlex – 2
@Orixim – 2
@Md-Knox – 2
@Iramtempus – 2
@HasieEST – 2
@ypoptos – 1
@thom4466 – 1
@theiodes – 1
@syssneck – 1
@stoudtlr – 1
@mythyrylfalcon – 1
@mr-polar-bear – 1
@mirotvorci – 1
@kuzaan – 1
@ishidinho – 1
@iEmpress – 1
@hugoantunes19 – 1
@flatronek – 1
@dkmnllth – 1
@diminian – 1
@dieing4me – 1
@coyote86 – 1
@anglerf1sh – 1
@Whoiskh – 1
@Vertazwar – 1
@Trajor – 1
@Steelcold – 1
@SgtRumpsteak – 1
@Scapulla – 1
@Sasnicka – 1
@Santaria94 – 1
@Rinias – 1
@Ovesss – 1
@OMGhixD-OG – 1
@Neversong97 – 1
@Lludara – 1
@Lessarcy – 1
@Lekiza – 1
@Korghan905 – 1
@Keerp96 – 1
@HazeMalaise – 1
@Hacki95 – 1
@Greyminder – 1
@Gizmoh – 1
@Frxd1234 – 1
@Evaynthelock – 1
@EllieSca – 1
@DobleInfery – 1
@Chundelac – 1
@ChromieShape – 1
@Asthalor – 1
@8Hoodie – 1

**Comment by ntrna** [comment]
@FrancescoBorzi, I don't have access to the discord, but I wonder if you could award my points manually, as you did last month. My CC account is Ntrn.

**Reply by FrancescoBorzi** [reply]
done
